### PR TITLE
Gradle 8.2

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,2 @@
 org.gradle.caching=true
 org.gradle.parallel=true
-org.gradle.vfs.watch=true
```
Comment (TWiStErRob) [inline gradle.properties:3]: I was wondering why, found it:
https://docs.gradle.org/7.0/release-notes.html#file-system-watching

**Review by TWiStErRob — APPROVED** [review]
Nice one! No deprecations? `--warning-mode=fail`

File: gradle.properties (hunk 1 above)
Comment (Goooler) [reply gradle.properties:3]: Yep, it's the default.

**Comment by TWiStErRob** [comment]
There are deprecations: https://scans.gradle.com/s/ibikx7qmmfecg/deprecations?expanded=WyI0Il0 we need to add platform-launcher to testRuntimeOnly probably.

**Comment by TWiStErRob** [comment]
Closes #235

**Comment by szpak** [comment]
```
A problem occurred configuring root project 'sample'. 
     Lock file: 
D:\a\publish-plugin\publish-plugin\build\tmp\compatTestJava11Gradle6.0.1\work\.gradle-test-kit\caches\6.0.1\generated-gradle-jars\generated-gradle-jars.lock
```
related to #228 :-/

**Comment by szpak** [comment]
Thanks for the PR @Goooler . Thanks @TWiStErRob for the review.
The Kotlin DSL is nicer and nicer (in a few years, maybe it will look similarly good to the Groovy one ;-)  ).

**Review by szpak — APPROVED** [review]

**Pull request merged**